src/components/Notifications/NotificationList/NotificationListItem/NotificationListItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import CandleIcon from "../../../CandleIcon/CandleIcon";
+
+function timeSince(date) {
```

Comment:
This function is just a copy of the util function, so we can just import it at the top of this file.